packages/ui/src/lib/tooltip/Tooltip.svelte

```diff
@@ -51,6 +51,7 @@
 
 <script lang="ts">
 export let tip: string | undefined = undefined;
+export let tipWidth = '';
```

Comment:
I would not call it tipWidth here because you're just passing extra class properties

so use `{$$props.class}` and in the parent component, just use class="your extra attributes"